docs/Makefile

```diff
@@ -51,6 +51,11 @@ examples:
 	rsync -a --exclude-from=.examplesignore ../samples/ts/ts-valueentity-counter/ "${managed_examples}/ts-valueentity-counter/"
 	rsync -a --exclude-from=.examplesignore ../samples/ts/ts-eventsourced-shopping-cart/ "${managed_examples}/ts-eventsourced-shopping-cart/"
 
+bundles:
+	bin/bundle.sh --zip "${managed_attachments}/js-customer-registry-quickstart.zip" ../samples/js/js-customer-registry-quickstart
```

Comment:
Note the issue about nested dir https://github.com/lightbend/akkaserverless-java-sdk/issues/380

Reply:
Do we want to fix that first before merging this in?

Reply:
Updated with the same changes as @pvlugter did for Java